spec/datadog/tracing/contrib/qless/support/job.rb

```diff
@@ -25,8 +25,9 @@ def string
 RSpec.shared_context 'Qless job' do
   include Qless::WorkerHelpers
 
-  let(:host) { ENV.fetch('TEST_REDIS_HOST', '127.0.0.1') }
-  let(:port) { ENV.fetch('TEST_REDIS_PORT', 6379) }
+  let(:host) { ENV.fetch('TEST_REDIS_OLD_HOST', '127.0.0.1') }
+  let(:port) { ENV.fetch('TEST_REDIS_OLD_PORT', 6379) }
```

Comment:
Minor: For consistency, I suggest always making this a string

```suggestion
  let(:port) { ENV.fetch('TEST_REDIS_OLD_PORT', '6379') }
```